beacon-chain/blockchain/process_attestation_helpers.go

```diff
@@ -113,40 +112,7 @@ func (s *Service) verifyAttestation(ctx context.Context, baseState *stateTrie.Be
 	}
 	indexedAtt := attestationutil.ConvertToIndexed(ctx, a, committee)
 	if err := blocks.VerifyIndexedAttestation(ctx, baseState, indexedAtt); err != nil {
-		if err == helpers.ErrSigFailedToVerify {
```

Comment:
This code has been old back in v0.9. We haven't seen attestation fails to verify due to bad committee caching.
This was one of the spec feed back. Using that as a good opportunity to delete